README.md

```diff
@@ -710,19 +707,20 @@ let completionBlock: () -> Void = {
 let completionBlock: (() -> Void)? = nil
```

Comment:
If it's a local constant, you do need it. It only gets auto-initialized to `nil` if it is a property.